setup.py

```diff
@@ -24,5 +24,8 @@
         'Operating System :: OS Independent',
     ],
     #long_description=open('README.rst').read(),
+    install_requires=[
+        "markdown",
```

Comment:
Should we add markdown to requirements.txt in styleguideprimer?

Reply:
it will be installed along with this package, by being included here.